SECURITY.md

```diff
@@ -15,3 +15,6 @@ This email is an alias to a subset of the scikit-learn maintainers' team.
 If the security vulnerability is accepted, a patch will be crafted privately
 in order to prepare a dedicated bugfix release as timely as possible (depending
 on the complexity of the fix).
+
+You can also report security vulnerabilities to
+[tidelift](https://tidelift.com/security).
```

Comment:
I think we need to rephrase this to state that it's preferable to always report the problem to `security@scikit-learn.org`, e.g.:

```suggestion
In addition to sending the report by email, you can also report security
vulnerabilities to [tidelift](https://tidelift.com/security).
```